File: app/src/main/java/room/entity/Country.java
package room.entity;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "countries")
public class Country
{
    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "id")
    private long id;

    @ColumnInfo(name = "iso")
    private String iso;

    @ColumnInfo(name = "name")
    private String name;

    public Country(String iso, String name)
    {
        this.iso = iso;
        this.name = name;
    }

    public long getId()
    {
        return this.id;
    }

    public String getIso()
    {
        return this.iso;
    }

    public String getName()
    {
        return this.name;
    }

    public void setId(long id)
    {
        this.id = id;
    }
}

File: app/src/main/java/localhost/android/me/database/entity/Country.kt
package localhost.android.me.database.entity

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "countries")
class Country(
    @ColumnInfo(name = "iso") private var iso: String,
    @ColumnInfo(name = "name") private var name: String
)
{
    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "id")
    private var id: Long = 0L

    fun getId(): Long
    {
        return this.id
    }

    fun getIso(): String
    {
        return this.iso
    }

    fun getName(): String
    {
        return this.name
    }

    fun setId(id: Long)
    {
        this.id = id
    }
}
